WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderPostAdapter.java

```diff
@@ -785,6 +788,8 @@ private void showComments(final ReaderPostViewHolder holder, final ReaderPost po
         if (canShowComments) {
             holder.commentCount.setCount(post.numReplies);
             holder.commentCount.setVisibility(View.VISIBLE);
+            holder.commentCount.setContentDescription(holder.cardView.getContext().getString(R.string.theme_view) + " "
```

Comment:
Similar to my previous comment, is it locale-friendly to append the strings like this? I'm also not wild about using `R.string.theme_view` here since that could be changed in a future themes redesign.

How about simply updating the layout XML to use "View comments?"